tests/integration/ha_tests/helpers.py

```diff
@@ -259,7 +259,7 @@ async def get_process_pid(
         return_code == 0
     ), f"Failed getting pid, unit={unit_name}, container={container_name}, process={process}"
 
-    stripped_pid = pid.strip()
+    stripped_pid = pid.strip().replace("\r", "").replace("\n", "")
```

Comment:
```suggestion
    stripped_pid = pid.strip().replace("\r", " ").replace("\n", " ")
```
I think in such case “--” replace will not be needed below

Reply:
This `mongod_cmd.replace("--", " --")` is needed because `cat /proc/{pid}/cmdline` returns this string
```
mongod--bind_ip_all--replSet=mongodb-k8s--dbpath=/var/lib/mongodb--logpath=/var/lib/mongodb/mongodb.log--auth--clusterAuthMode=keyFile--keyFile=/etc/mongod/keyFile
```

Reply:
This change was maid to prevent exceptions on int(stripped_pid)